src/index.js

```diff
@@ -80,25 +82,51 @@ function displayAllTrips() {
 
 function toggleFormView() {
   document.querySelector('.welcome-message').classList.toggle('hidden');
-  document.querySelector('.plan-trip-button').classList.toggle('hidden');
+  planTripButton.classList.toggle('hidden');
   document.querySelector('.new-trip-form').classList.toggle('hidden');
+  bookTripButton.classList.toggle('hidden');
```

Comment:
For refactoring, could consider adding an additional class that can group together elements that hide/unhide in tandem and creating an array object.

Reply:
Comment was unclear: something like`const formItems = Array.from(document.querySelectorAll('.form-item')` and passing those objects through a function. May be overkill considering the limited amount of groups there would be.